<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_COAS_StateNodePart</class>
 <widget class="QWidget" name="P_COAS_StateNodePart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">
font-family:&quot;微软雅黑&quot;;
font-size:20px;
font-weight:bold;
padding:4px;
color:#ffffff;
border-radius:3px;
border:1px solid #000000;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #444444, stop:0.3 #525252, stop:0.9 #525252, stop:1 #444444 )</string>
              </property>
              <property name="text">
               <string>状态节点</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_relationTable">
              <property name="text">
               <string>查看关系表</string>
              </property>
              <property name="icon">
               <iconset resource="../../../drillgifactionsequenceeditor.qrc">
                <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/menu/Common_SelectAll.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/menu/Common_SelectAll.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="styleSheet">
               <string notr="true">
QTableView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>参数</string>
              </property>
              <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,1,3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>节点名称</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_name"/>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>节点优先级</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>节点标签</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_tagTank">
                 <property name="text">
                  <string>[]</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QSpinBox" name="spinBox_priority">
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>节点权重</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>可被动作元打断</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>备注</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QCheckBox" name="checkBox_canBeInterrupted">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QSpinBox" name="spinBox_proportion">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButton_tagTank">
                 <property name="text">
                  <string>编辑标签</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1" colspan="3">
                <widget class="QPlainTextEdit" name="plainTextEdit_note">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>播放</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="4" column="1">
                <widget class="QPushButton" name="pushButton_moveToLeft">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&lt;&lt;</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="5" column="1">
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="0" rowspan="4">
                <widget class="QTableWidget" name="tableWidget_left">
                 <property name="styleSheet">
                  <string notr="true">
QTableView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}</string>
                 </property>
                 <column>
                  <property name="text">
                   <string>已有状态元</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="pushButton_moveToRight">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="4">
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>播放方式</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_playType">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>随机播放状态元</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>顺序播放状态元</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>随机播放嵌套集合</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>顺序播放嵌套集合</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="2" rowspan="4">
                <widget class="QTableWidget" name="tableWidget_right">
                 <property name="styleSheet">
                  <string notr="true">
QTableView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}</string>
                 </property>
                 <column>
                  <property name="text">
                   <string>当前配置</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item row="2" column="3" rowspan="4">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pushButton_moveUp">
                   <property name="text">
                    <string>上移</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_moveDown">
                   <property name="text">
                    <string>下移</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_editConfig">
                   <property name="text">
                    <string>手动配置</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Line" name="line">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_checkLoop">
                   <property name="text">
                    <string>检查嵌套关系</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_relationTable_2">
                   <property name="text">
                    <string>查看关系表</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="1" column="2" colspan="2">
                <widget class="QWidget" name="widget_randomMax" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>随机播放的次数上限</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_randomMax">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../drillgifactionsequenceeditor.qrc"/>
 </resources>
 <connections/>
</ui>
